Source/WebCore/Modules/fetch/FetchHeaders.h

```diff
@@ -38,20 +38,20 @@ namespace WebCore {
 
 class ScriptExecutionContext;
 
+enum class FetchHeadersGuard : uint8_t {
+    None,
+    Immutable,
+    Request,
+    RequestNoCors,
+    Response
+};
+
 class FetchHeaders : public RefCounted<FetchHeaders> {
 public:
-    enum class Guard {
```

Comment:
Instead of removing it, we could do using Guard = FetchHeadersGuard.
That would remove the need of some of the changes

Reply:
Thanks for the tip. PR is now **MUCH** cleaner.